Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.yml

```
@@ -762,17 +774,857 @@ script:
+      description: 'Latest status of the nonCustodialDataSource. Possible values are: Active, Released.'
+      type: String
+    - contextPath: MsGraph.NoncustodialDataSource.DisplayName
+      description: 'Display name of the noncustodialDataSource.'
+      type: String
+    - contextPath: MsGraph.NoncustodialDataSource.HoldStatus
+      description: 'The hold status of the nonCustodialDataSource.The possible values are: notApplied, applied, applying, removing, partial'
+      type: String
+    - contextPath: MsGraph.NoncustodialDataSource.LastModifiedDateTime
+      description: 'Last modified date and time of the nonCustodialDataSource.'
+      type: Date
+    - contextPath: MsGraph.NoncustodialDataSource.ReleasedDateTime
+      description: 'Date and time that the nonCustodialDataSource was released from the case.'
+      type: Date
+  - arguments:
+    - description: The id of the eDiscovery case.
+      isArray: false
+      name: case_id
+      required: true
+    - description: A comma seperated list of custodians ids to apply a hold to.
+      isArray: true
+      name: custodian_id
+      required: true
+    description: |-
+      Start the process of applying hold on eDiscovery custodians.
+      Available return statuses: 
+      notApplied - The custodian is not on Hold (all sources in it are not on hold).
+      applied - The custodian is on Hold (all sources are on hold).
+      applying - The custodian is in applying hold state (applyHold operation triggered).
+      removing - The custodian is in removing the hold state(removeHold operation triggered).
+      partial - The custodian is in mixed state where some sources are on hold and some not on hold or error state.
+    execution: false
+    name: msg-apply-hold-ediscovery-custodian
+  - arguments:
+    - description: The id of the eDiscovery case.
+      isArray: false
+      name: case_id
+      required: true
+    - description: A comma seperated list of custodians ids to remove a hold from.
+      isArray: true
+      name: custodian_id
+      required: true
+    description: Start the process of removing hold from eDiscovery custodians.
+    execution: false
+    name: msg-remove-hold-ediscovery-custodian
+  - arguments:
+    - description: The id of the eDiscovery case.
+      isArray: false
+      name: case_id
+      required: true
+    - description: The display name of the search.
+      isArray: false
+      name: display_name
+      required: true
+    - description: The description of the search.
+      isArray: false
+      name: description
+      required: false
+    - description: The query string used for the search. The query string in KQL (Keyword Query Language) format. For details, see Keyword queries and search conditions for Content Search and eDiscovery. You can refine searches by using fields paired with values; for example, subject:"Quarterly Financials" AND Date>=06/01/2016 AND Date<=07/01/2016.
```

```suggestion
    - description: The query string used for the search. The query string format is KQL (Keyword Query Language). For details, see Keyword queries and search conditions for Content Search and eDiscovery. You can refine searches by using fields paired with values, for example, subject:"Quarterly Financials" AND Date>=06/01/2016 AND Date<=07/01/2016.
```

Where is this:  Keyword queries and search conditions for Content Search and eDiscovery?  Need a link or remove.